### ISSUE TITLE
Headers are not populated for binary-only bodies

### ISSUE THREAD

**Issue by jhillyerd** [jhillyerd/enmime#25]
envelope.go:func parseBinaryOnlyBody doesn't copy headers like From, Subject, To, etc into its Root part.

**Comment by jhillyerd** [comment]
When forcing isBinaryBody()->true, TestInlineTextBody() reported missing headers:

```go
&enmime.Envelope{
Text:"",
HTML:"",
Root:(*enmime.Part)(0xc4202f1e00),
Attachments:[]*enmime.Part(nil),
Inlines:[]*enmime.Part{(*enmime.Part)(0xc4202f1d40)},
OtherParts:[]*enmime.Part(nil),
Errors:[]*enmime.Error(nil),
header:(*textproto.MIMEHeader)(0xc4202f1d40)}
```
```
--- FAIL: TestInlineTextBody (0.00s)
        envelope_test.go:946: Got Text with value , wanted value:
                Just some html content
        envelope_test.go:949: Expected "" to contain "Just some html content"
        envelope_test.go:954: Failed to extract expected headers. Got 0 headers, expected 9
        envelope_test.go:959: Header named "Subject" was missing, want it to exist
        envelope_test.go:959: Header named "Content-Type" was missing, want it to exist
        envelope_test.go:959: Header named "Content-Transfer-Encoding" was missing, want it to exist
        envelope_test.go:959: Header named "Date" was missing, want it to exist
        envelope_test.go:959: Header named "To" was missing, want it to exist
        envelope_test.go:959: Header named "Message-Id" was missing, want it to exist
        envelope_test.go:959: Header named "Mime-Version" was missing, want it to exist
        envelope_test.go:959: Header named "From" was missing, want it to exist
        envelope_test.go:959: Header named "Content-Disposition" was missing, want it to exist
```

**Comment by jhillyerd** [comment]
Looks like this was fixed in #30 - but I will add a more detailed unit test.
